src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's Contact.
+ * Parent class for Address, Phone and Email
```

Comment:
Don't include the subclasses in a super class. You never know who will use this super class in future.